ios/Classes/PurchasesFlutterPlugin.m

```diff
@@ -146,6 +146,9 @@ - (void)handleMethodCall:(FlutterMethodCall *)call
         [self setCreative:creative result:result];
     } else if ([@"collectDeviceIdentifiers" isEqualToString:call.method]) {
         [self collectDeviceIdentifiersWithResult:result];
+    } else if ([@"canMakePayments" isEqualToString:call.method]) {
+          NSArray *features = arguments[@"features"];
```

Comment:
I didn't catch it when we were doing the `purchases-hybrid-common`, but I think best practice here would have been to define the type as `NSArray <NSString *>*`. 
It doesn't really matter for now, though, since the array isn't necessary in the first place.

Reply:
well that one isn't merged yet either :)